zhaquirks/tuya/mcu/__init__.py

```diff
@@ -753,7 +753,7 @@ def __init__(
 
     def battery(
         self,
-        dp_id: int,
+        dp_id: int = 15,
```

Comment:
I think we don't even need a default for this, even if we always pass it in as a kwarg.
That said, if the datapoints for battery/soil/temp are mostly/always the same, the defaults might still make sense. But I also see why you don't want to hide everything in defaults.